packages/angular/package.json

```diff
@@ -14,33 +14,30 @@
   },
   "private": true,
   "devDependencies": {
-    "@angular/animations": "^17.2.0",
-    "@angular/common": "^17.2.0",
-    "@angular/compiler": "^17.2.0",
-    "@angular/core": "^17.2.0",
-    "@angular/forms": "^17.2.0",
-    "@angular/platform-browser": "^17.2.0",
-    "@angular/platform-browser-dynamic": "^17.2.0",
-    "@angular/router": "^17.2.0",
-    "@angular-devkit/build-angular": "^17.3.8",
-    "@angular-eslint/builder": "17.5.3",
-    "@angular-eslint/eslint-plugin": "17.5.3",
-    "@angular-eslint/eslint-plugin-template": "17.5.3",
-    "@angular-eslint/schematics": "17.5.3",
-    "@angular-eslint/template-parser": "17.5.3",
+    "@angular-devkit/build-angular": "^18.2.10",
+    "@angular-eslint/builder": "18.4.0",
+    "@angular-eslint/eslint-plugin": "18.4.0",
+    "@angular-eslint/eslint-plugin-template": "18.4.0",
+    "@angular-eslint/schematics": "18.4.0",
+    "@angular-eslint/template-parser": "18.4.0",
+    "@angular/animations": "^18.2.9",
     "@angular/cli": "^17.2.2",
-    "@angular/compiler-cli": "^17.2.0",
-    "@typescript-eslint/eslint-plugin": "7.18.0",
-    "@typescript-eslint/parser": "7.18.0",
+    "@angular/common": "^18.2.9",
+    "@angular/compiler": "^18.2.9",
+    "@angular/compiler-cli": "^18.2.9",
+    "@angular/core": "^18.2.9",
+    "@angular/forms": "^18.2.9",
+    "@angular/platform-browser": "^18.2.9",
+    "@angular/platform-browser-dynamic": "^18.2.9",
+    "@angular/router": "^18.2.9",
+    "@typescript-eslint/eslint-plugin": "7.11.0",
+    "@typescript-eslint/parser": "7.11.0",
     "eslint": "^8.57.0",
     "jest-preset-angular": "^14.2.2",
-    "ng-packagr": "^17.3.0",
+    "ng-packagr": "^18.2.1",
     "rxjs": "~7.8.0",
     "tslib": "^2.3.0",
-    "zone.js": "~0.14.3",
-    "typescript": "^4.7.4"
-  },
-  "overrides": {
-    "typescript": "^4.7.4"
```

Comment:
Why is this no longer needed?

Reply:
Because we have the override already in the root `package.json`.